packages/phoenix-event-display/src/loaders/phoenix-loader.ts

```diff
@@ -290,6 +290,16 @@ export class PhoenixLoader implements EventDataLoader {
       );
     }
 
+    if (eventData.Fibers) {
```

Comment:
We already have an [`LHCbLoader`](https://github.com/HSF/phoenix/blob/master/packages/phoenix-event-display/src/loaders/lhcb-loader.ts) which this code should belong to.

Since we are already extending the `LHCbLoader` from `PhoenixLoader`, what you can do is put the method there, and override the [`loadObjectTypes`](https://github.com/HSF/phoenix/blob/b1a88476b5a73412f95f0360bdad4c099bed917d/packages/phoenix-event-display/src/loaders/phoenix-loader.ts#L145) method on `LHCbLoader`, call the parent version of it and then add your LHCb specific code there.

The `CMSLoader` does exactly the same [here](https://github.com/HSF/phoenix/blob/master/packages/phoenix-event-display/src/loaders/cms-loader.ts#L28). For CMS, we are adding `MuonChambers` as a custom object.

Now that I wrote this comment, I just realized that I wrote a dedicated guide for this. If you just read through this, you will have a good idea of what to do: <https://github.com/HSF/phoenix/blob/master/guides/developers/event-data-loader.md>

It will look something like this in the `LHCbLoader`:

```ts
  protected loadObjectTypes(eventData: any) {
    // Load all other object types.
    super.loadObjectTypes(eventData);

    // Load LHCb specific object types.
    if (eventData.Fibers) {
      this.addObjectType(
        eventData.Fibers,
        LHCbObjects.getFibers,
        'Fibers',
        false
      );
    }
  }
```